test/assets/pdf.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>PDF</title>
+    <style>
+      @font-face {
+        font-family: 'custom-font';
+        font-display: swap;
+        src: url(https://custom-font.com/font.woff2) format('woff2');
```

Comment:
Is it possible to use a local font here instead of loading something from a third party?

Reply:
that's a good point, will give it a go